core/src/main/scala/org/apache/spark/h2o/utils/H2OContextRestAPIUtils.scala

```diff
@@ -52,6 +52,22 @@ trait H2OContextRestAPIUtils extends H2OContextUtils {
     getNodes(cloudV3)
   }
 
+  def verifyWebOpen(nodes: Array[NodeDesc], conf: H2OConf): Unit = {
+    val nodesWithoutWeb = nodes.flatMap { node =>
+      try {
+        getCloudInfoFromNode(node, conf)
+        None
+      } catch {
+        case _: H2OClusterNodeNotReachableException => Some(node)
+      }
+    }
+    if (nodesWithoutWeb.nonEmpty) {
+      throw new H2OClusterNodeNotReachableException(
+        s"The following worker nodes are not reachable, but belong to the cluster identified by " +
+          s"${conf.h2oCluster.get}:${nodesWithoutWeb.mkString("\n")}", null)
```

Comment:
`.mkString("[", ", ", "]")`?

Reply:
`mkString("\n")` is there to print each NodeDesc on new line. What would be output of `.mkString("[", ", ", "]")`?

Reply:
it would formated as an array. `[node1, node2, node3]`. I like the idea of having a node on separete line, but shouldn't we do the same with `conf.h2oCluster.get`?
Example:
```
mycluster
----------
node1
node2
node3
```

Reply:
I see, thanks for the mkString explanation! Yup, good idea, let me fix it

Reply:
<img width="701" alt="Screen Shot 2019-11-07 at 1 21 02 PM" src="https://user-images.githubusercontent.com/4374603/68388668-797e5780-0161-11ea-8e41-e593f4d3e1d0.png">

Looks like this at this moment, what do you think?

Reply:
Looks good to me!